scripts/ci_setup.py

```diff
@@ -315,28 +347,37 @@ def __main(args: list) -> int:
     perfHash = decoded_output if args.get_perf_hash else args.perf_hash
 
     framework = ChannelMap.get_target_framework_moniker(args.channel)
+
+    extension = ".cmd" if args.target_windows else ".sh"
+    output_file = args.output_file + extension
+
     if not framework.startswith('net4'):
         target_framework_moniker = dotnet.FrameworkAction.get_target_framework_moniker(framework)
         dotnet_version = dotnet.get_dotnet_version(target_framework_moniker, args.cli) if args.dotnet_versions == [] else args.dotnet_versions[0]
         commit_sha = dotnet.get_dotnet_sdk(target_framework_moniker, args.cli) if args.commit_sha is None else args.commit_sha
-        if(args.commit_time is not None):
-            try:
-                parsed_timestamp = datetime.datetime.strptime(args.commit_time, '%Y-%m-%d %H:%M:%S %z').astimezone(datetime.timezone.utc)
-                source_timestamp = parsed_timestamp.strftime('%Y-%m-%dT%H:%M:%SZ')
-            except ValueError:
-                getLogger().warning('Invalid commit_time format. Please use YYYY-MM-DD HH:MM:SS +/-HHMM. Attempting to get commit time from api.github.com.')
-                source_timestamp = dotnet.get_commit_date(target_framework_moniker, commit_sha, repo_url)
-        else:
-            source_timestamp = dotnet.get_commit_date(target_framework_moniker, commit_sha, repo_url)
+        # TODO: Fix this
+        source_timestamp = datetime.datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%SZ')
+
+        # if(args.commit_time is not None):
```

Comment:
Reminder to uncomment this and figure out what to do for commit time for local runs.